### PR TITLE
feat(card): webcomponent

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Web component version of card. Also updated the variable for card body color.

**Solving issue**  
Fixes: [AB#3147](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3147)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Card -> WebComponent
3. Try out the controls and check that the design is correct.

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

**Screenshots**  

**Additional context**

### CODE REVIEW

**Comment by EmelieLitwin** [comment]
This one I feel we should discuss on a team checkin or at least with @charlottelorensson on how to present in storybook and maybe if there are some things that could be improved. There are sooo many examples in UI-kit

**Comment by theJohnnyMe** [comment]
If the "Header image" control is empty, the Storybook will render missing/broken image:  
Screenshot 2023-02-03 at 10 16 17

File: tegel/src/components/card/sdds-card.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  @Prop() bodyDivider: boolean = false;
+
+  /** Makes the card clickable. */
+  @Prop() clickable: boolean = false;
+
+  /** ID for the card, must be unique.
+   *
+   * **NOTE**: If you're listening for card events you need to set this ID yourself to identify the card, as the default ID is random and will be different every time.
+   */
+  @Prop() cardId: string = crypto.randomUUID();
```
Comment (theJohnnyMe) [inline tegel/src/components/card/sdds-card.tsx:43]: Nice that you take care of this, I remember a conversation from the data table we had. Glad to see that is becoming "a standard" for our components ;)

File: tegel/src/components/card/sdds-card.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  @Prop() bodyImg: string;
+
+  /** Alt text for the body image */
+  @Prop() bodyImgAlt: string;
+
+  /** Divider for the body */
+  @Prop() bodyDivider: boolean = false;
+
+  /** Makes the card clickable. */
+  @Prop() clickable: boolean = false;
```
Comment (theJohnnyMe) [inline tegel/src/components/card/sdds-card.tsx:37]: Cool to see this as a feature + you took care of event firing and ID.

File: tegel/src/components/card/sdds-card.scss [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    color: var(--sdds-card-sub-headline);
+  }
+
+  .card-body-img {
+    width: 100%;
+  }
+
+  .sdds-divider {
+    margin: 0 16px;
+    margin-top: 16px;
```
Comment (theJohnnyMe) [inline tegel/src/components/card/sdds-card.scss:63]:
```suggestion
    margin: 16px 16px 0;
```

**Review by theJohnnyMe — APPROVED** [review]
I saw you fixed the "Header image" issue very quickly. 
Other things look good to me. Nice approach for making slots so we give more freedom to users when it comes to passing content.

I saw a comment from @EmelieLitwin about talking about this with @charlottelorensson. I leave it up to you to check with them if something more was needed.  Maybe you resolve it in a meantime.

**Pull request merged**